pkg/policy/execute.go

```diff
@@ -252,7 +254,8 @@ func (e *Executor) ExecutePolicies(ctx context.Context, req *ExecuteRequest, pol
 		}
 	}
 	if !found && len(selector) > 0 {
-		return nil, errPolicyOrQueryNotFound
+		e.log.Error("policy not found with provided selector", "selector", selector, "policy names", pnames)
+		return nil, fmt.Errorf("policy not found with provided selector: %s", selector)
```

Comment:
I think leaving this to be `errPolicyOrQueryNotFound` is better since the policy description PR takes this and prepends the selector to it so it's still visible and there's error typing.

Reply:
Ok. I will change it back